espnet_model_zoo/table.csv

```diff
@@ -151,3 +151,4 @@ ljspeech,tts,kan-bayashi/ljspeech_tts_train_joint_conformer_fastspeech2_hifigan_
 ljspeech,tts,kan-bayashi/ljspeech_tts_finetune_joint_conformer_fastspeech2_hifigan_raw_phn_tacotron_g2p_en_no_space_train.total_count.ave,https://zenodo.org/record/5498896/files/tts_finetune_joint_conformer_fastspeech2_hifigan_raw_phn_tacotron_g2p_en_no_space_train.total_count.ave.zip?download=1,22050,en,female,1.7.1,0.10.3a2,628b46,true
 ljspeech,tts,kan-bayashi/ljspeech_joint_train_conformer_fastspeech2_hifigan,https://zenodo.org/record/5498487/files/tts_train_joint_conformer_fastspeech2_hifigan_raw_phn_tacotron_g2p_en_no_space_train.total_count.ave.zip?download=1,22050,en,female,1.7.1,0.10.3a2,628b46,true
 ljspeech,tts,kan-bayashi/ljspeech_joint_finetune_conformer_fastspeech2_hifigan,https://zenodo.org/record/5498896/files/tts_finetune_joint_conformer_fastspeech2_hifigan_raw_phn_tacotron_g2p_en_no_space_train.total_count.ave.zip?download=1,22050,en,female,1.7.1,0.10.3a2,628b46,true
+tedlium2,asr,D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave,https://huggingface.co/api/models/D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave,16000,en,,,,,true
```

Comment:
@D-Keqi , @sw005320 , I found the issue and it is not related to espnet or espnet_zoo. Instead of using `https://huggingface.co/api/models/D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave` as URL, use just `huggingface.co`.
This is the default value required for the ESPnet frontends to get that the model belongs to huggingface.

But, You will get a different error:
```python
File "/export/db/espnet/espnet_model_zoo/ci/test_model.py", line 16, in _asr
    nbests = speech2text(speech)
  File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 28, in decorate_context
    return func(*args, **kwargs)
  File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/espnet2/bin/asr_inference.py", line 216, in __call__
    enc, _ = self.asr_model.encode(**batch)
  File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/espnet2/asr/espnet_model.py", line 244, in encode
    assert encoder_out.size(1) <= encoder_out_lens.max(), (
AttributeError: 'NoneType' object has no attribute 'max'
```
that is not related to this but to a None output, not sure if has to do with the streaming model. tried to increase the number of samples but it did not work.

Reply:
> @D-Keqi , @sw005320 , I found the issue and it is not related to espnet or espnet_zoo. Instead of using `https://huggingface.co/api/models/D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave` as URL, use just `huggingface.co`. This is the default value required for the ESPnet frontends to get that the model belongs to huggingface.
> 
> But, You will get a different error:
> 
> ```python
> File "/export/db/espnet/espnet_model_zoo/ci/test_model.py", line 16, in _asr
>     nbests = speech2text(speech)
>   File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/torch/autograd/grad_mode.py", line 28, in decorate_context
>     return func(*args, **kwargs)
>   File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/espnet2/bin/asr_inference.py", line 216, in __call__
>     enc, _ = self.asr_model.encode(**batch)
>   File "/utils/python3.9/envs/zoo/lib/python3.9/site-packages/espnet2/asr/espnet_model.py", line 244, in encode
>     assert encoder_out.size(1) <= encoder_out_lens.max(), (
> AttributeError: 'NoneType' object has no attribute 'max'
> ```
> 
> that is not related to this but to a None output, not sure if has to do with the streaming model. tried to increase the number of samples but it did not work.

Thanks! We should use the asr_inference_streaming.py https://github.com/espnet/espnet/blob/master/espnet2/bin/asr_inference_streaming.py for inference. I agree with you, this issue is related to streaming decoding.

Reply:
Given this case, I will recommend you to modify the task. Instead of `asr` use `asr_stream` and add the test for asr_stream at https://github.com/espnet/espnet_model_zoo/blob/master/ci/test_model.py

Reply:
Your new line should be smth like:

`
tedlium2,asr_stream,D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave,huggingface.co,16000,en,,,,,true

Reply:
> Your new line should be smth like:
> 
> ` tedlium2,asr_stream,D-Keqi/espnet_asr_train_asr_streaming_transformer_raw_en_bpe500_sp_valid.acc.ave,huggingface.co,16000,en,,,,,true`

I see, Thanks a lot!

Reply:
It seems that I still have a CI issue, I am wondering if the system uses the old test_mode.py for CI testing? or uses the new test_model.py I have modified? @Fhrozen